cordova.conf

```diff
@@ -110,6 +108,13 @@ from buildbot.config import BuilderConfig
 from buildbot.process.factory import BuildFactory
 from buildbot.steps.transfer import FileDownload
 
+def RMCommand(path, **kwargs):
+    js_script = "var s = require('shelljs'); console.log('removing {path}'); s.rm('-rf', '{path}');".format(path=path)
+    return ShellCommand(command=['node', '-e', js_script], **kwargs)
+
+def InstallShellJS():
```

Comment:
Another way is to note shelljs as a prerequisite for build slave (like the grunt-cli (https://github.com/apache/cordova-medic#install-prerequisites))